includes/class-wc-siftscience-html.php

```diff
@@ -109,7 +109,7 @@ public function create_element( $type, $id, $label = '', $desc = '', $element_op
 
 			switch ( $type ) {
 				case self::WC_CUSTOM_ELEMENT:
-					$type = 'wc_sift_' . $id; // this is the custom type name needed by WooCommerce.
```

Comment:
If you want, you can replace this like with:

```
$type = $id;
```

This will also work. I only added `wc_sift_` for extra clarity/distinction in the generated hook name.